tests/ae-inventory-subjects-itest/src/test/java/org/metaeffekt/core/itest/common/download/WebAccess.java

```diff
@@ -96,6 +97,8 @@ public InputStream fetchResponseBodyFromUrlAsInputStream(URL url, Map<String, St
                 }
 
                 final HttpResponse response = httpClient.execute(httpGet);
+
+                if(response.getStatusLine().getStatusCode() != 200) throw new IllegalArgumentException();
```

Comment:
We should also supply a SHA-256 checksum for every loaded file to insure that we get the right thing.